libgc/include/private/gc_priv.h

```diff
@@ -1330,7 +1330,7 @@ void GC_initiate_gc GC_PROTO((void));
   				/* If the mark state is invalid, this	*/
   				/* becomes full colleection.  Otherwise */
   				/* it's partial.			*/
-void GC_push_all GC_PROTO((ptr_t bottom, ptr_t top));
+void GC_push_all (void* bottom, void* top);
```

Comment:
Mainline boehm is considering this change also. https://github.com/ivmai/bdwgc/pull/204

Reply:
Mainline boehm is now valid C++.